docs/modules/installation/pages/upgrading.adoc

```diff
@@ -45,5 +47,38 @@ perl -pi -e 's/opennms-grafana-plugin-fault-datasource/opennms-grafana-plugin-en
 
 . Publish your changes to your Grafana instance:
 +
-[source, console]
+[source, shell]
 wizzy export datasources
+
+== Upgrade dashboards for version 9.x
+
+When you upgrade your {product-name} installation to version 9.x, you must upgrade any previously existing dashboards.
+If they are not upgraded, they will not be displayed.
+
+{product-name} version 9.x includes a dashboard conversion tool to upgrade dashboards created using previous versions.
+To use the tool, you must export your dashboards from the previous version of the plugin.
```

Comment:
It's not clear to me that the export has to be done from the previous version. Please run this by @synqotik to be sure.

Reply:
I'll specify this in an admonition. Thanks for clarifying!

Reply:
@mmahacek 's comment is correct, you need to export from a previous version first, then use the conversion tool.

One note, if you did an upgrade of the plugin, then tried to open an existing/previous dashboard, it would display (panels would display in their proper places) but with a lot of errors and data loss (queries would appear to be empty, etc.). So, I think the wording `"If they are not upgraded, they will not be displayed"` is probably good to keep, it will avert user surprise/frustration.